nni/retiarii/evaluator/pytorch/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .base import PyTorchImageClassificationTrainer, PyTorchMultiModelTrainer
```

Comment:
why not change `Trainer` to `Evaluator`?

Reply:
The base file is deprecated. These are not public APIs.